implementation/elasticsearch/src/main/java/net/opentsdb/meta/NamespacedAggregatedDocumentSchema.java

```diff
@@ -107,6 +115,13 @@ public String type() {
       tsdb.getConfig().register(getConfigKey(CLIENT_ID), null, false, "Meta client id.");
     }
 
+    String skip_meta_namespaces = tsdb.getConfig().getString(SKIP_META_NAMESPACES);
```

Comment:
Register this as a YAML map with keys as the namespace and values as the retention and lets make it dynamic so that we can tweak the config on the fly.